jenkins/databricks/clusterutils.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+import json
+import time
+import os
+import requests
+import sys
+
+class ClusterUtils(object):
+
+    @staticmethod
+    def generate_create_templ(sshKey, initScriptFile, version):
+        # TODO (in followup) - create directory if not there
+        init_script_loc = 'dbfs:/databricks/ci_init_scripts/' + initScriptFile
+        dt = datetime.now()
+        databricks_runtime_version = '7.0.x-gpu-ml-scala2.12'
+        idle_timeout = 240
```

Comment:
nit: Might be good to have a follow on so that it is tied to the timeout for the Jenkins job so we don't have a config for one timeout and a separate hard coded timeout.  I really hope we never get over 4 hours to run the tests, but it could also save us some money in the case of an error.

Reply:
I'll just make it a parameter now